services/wallet/token/tokenstore.go

```diff
@@ -32,1665 +33,1869 @@ func GetTokenPegSymbol(symbol string) string {
-				Name:     "DAI Stablecoin",
-				Symbol:   "DAI",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  42161,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f"),
-				Name:     "Wrapped BTC",
-				Symbol:   "WBTC",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  42161,
-			},
-			&Token{
-				Address:  common.HexToAddress("0xfa7f8980b0f1e64a2062791cc3b0871572f1f7f0"),
-				Name:     "Uniswap",
-				Symbol:   "UNI",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  42161,
-			},
-			&Token{
-				Address:  common.HexToAddress("0xf97f4df75117a78c1a5a0dbb814af92458539fb4"),
-				Name:     "ChainLink Token",
-				Symbol:   "Link",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  42161,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x17fc002b466eec40dae837fc4be5c67993ddbd6f"),
-				Name:     "Frax",
-				Symbol:   "FRAX",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  42161,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x4d15a3a2286d883af0aa1b3f21367843fac63e07"),
-				Name:     "True USD",
-				Symbol:   "TUSD",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  42161,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x23a941036ae778ac51ab04cea08ed6e2fe103614"),
-				Name:     "Graph Token",
-				Symbol:   "GRT",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  42161,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x680447595e8b7b3aa1b43beb9f6098c79ac2ab3f"),
-				Name:     "Decentralized USD",
-				Symbol:   "USDD",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  42161,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x11cdb42b0eb46d95f990bedd4695a6e3fa034978"),
-				Name:     "Curve DAO Token",
-				Symbol:   "CRV",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  42161,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x17078F231AA8dc256557b49a8f2F72814A71f633"),
-				Name:     "USD Coin",
-				Symbol:   "USDC",
-				Color:    "#f8f8f8",
-				Decimals: 6,
-				ChainID:  421613,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x265B25e22bcd7f10a5bD6E6410F10537Cc7567e8"),
-				Name:     "Tether USD",
-				Symbol:   "USDT",
-				Color:    "#f8f8f8",
-				Decimals: 6,
-				ChainID:  421613,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x3e622317f8c93f7328350cf0b56d9ed4c620c5d6"),
-				Name:     "Dai Stablecoin",
-				Symbol:   "DAI",
-				Color:    "#f8f8f8",
-				Decimals: 18,
-				ChainID:  11155111,
-			},
-			&Token{
-				Address:  common.HexToAddress("0x44a739916D41eC0226d98F83BE5364B69078DA41"),
-				Name:     "USD Coin",
-				Symbol:   "USDC",
-				Color:    "#f8f8f8",
-				Decimals: 6,
-				ChainID:  421614,
+		TokenListID: []*Token{
+			&Token{
+				Address:     common.HexToAddress("0x6b175474e89094c44da98b954eedeac495271d0f"),
+				Name:        "Dai Stablecoin",
+				Symbol:      "DAI",
+				Color:       "#f8f8f8",
```

Comment:
@alaibe can we remove this color property? it has no significance for tokens right

Reply:
yes we can